cli/flox/doc/flox.md

```diff
@@ -85,201 +42,89 @@ set -px XDG_DATA_DIRS "/nix/var/nix/profiles/default"
+## Sharing Commands
 
+`push`
+:   Send an environment to FloxHub.
 
-**publish**
-:   Perform a build, (optionally) copy binaries to a cache,
-    and add package metadata to a flox channel.
+`pull`
+:   Pull an environment from FloxHub.
 
-**run**
-:   Run flake application from the requested package (or "installable").
+## Additional Commands
 
-## Administration
+`update`
+:   Update an environment's base catalog or update the global base catalog.
 
-**config** [ (--list|-l) (--confirm|-c) (--reset|-r) ]
-:   Configure and/or display user-specific parameters.
+`upgrade`
+:   Upgrade packages in an environment.
 
-**git** `<git-subcommand>` [ `<args>` ]
-:   Direct access to git command invoked in the `floxmeta` repository clone.
+`containerize`
+:   Export a Flox environment as a container image.
 
-**gh** `<gh-subcommand>` [ `<args>` ]
-:   Direct access to gh command. For expert use only.
+`config`
+:   View and set configuration options.
 
-# PACKAGE ARGUMENTS
+`auth`
+:   FloxHub authentication commands.
 
-Flox package arguments are specified as a tuple of
-stability, channel, name, and version in the format:
-`<stability>`.`<channel>`.`<name>`@`<version>`
-
-The version field is optional, defaulting to the latest version if not specified.
-
-The stability field is also optional, defaulting to "stable" if not specified.
-
-The channel field is also optional, defaulting to "nixpkgs-flox" if not specified,
-_but only if using the "stable" stability_. If using anything other than the
-default "stable" stability, the channel *must* be specified.
-
-For example, each of the following will install the latest hello version 2.12 from
-the stable channel:
-```
-flox install stable.nixpkgs-flox.hello@2.12
-flox install stable.nixpkgs-flox.hello
-flox install nixpkgs-flox.hello@2.12
-flox install nixpkgs-flox.hello
-flox install hello@2.12
-flox install hello
-```
+# ENVIRONMENT VARIABLES
 
-... and each of the following will install the older hello version 2.10
-from the stable channel:
-```
-flox install stable.nixpkgs-flox.hello@2.10
-flox install nixpkgs-flox.hello@2.10
-flox install hello@2.10
-```
+`$FLOX_PROMPT_ENVIRONMENTS`
+:   Contains a space-delimited list of the active environments,
+    e.g. 'owner1/foo owner2/bar local_env'.
 
-... but only the following will install the older hello version 2.10 from the unstable channel:
-```
-flox install unstable.nixpkgs-flox.hello@2.10
-```
+`$_FLOX_ACTIVE_ENVIRONMENTS`
+:   A JSON array containing one object per active environment.
+    This is currently an implementation detail
+    and its contents are subject to change.
 
-# ENVIRONMENT VARIABLES
+`$FLOX_PROMPT_COLOR_{1,2}`
+:   Flox adds text to the beginning of the shell prompt to indicate which
+    environments are active.
+    A set of default colors are used to color this prompt,
+    but the colors may be overridden with the `$FLOX_PROMPT_COLOR_1` and
+    `$FLOX_PROMPT_COLOR_2` environment variables.
```

Comment:
question nonblocking: do you think `$FLOX_PROMPT_ENVIRONMENTS`, `$_FLOX_ACTIVE_ENVIRONMENTS`, and `$FLOX_PROMPT_COLOR_{1,2}` belong here or in `flox-activate.md`?

Reply:
I have a slight preference to move `EDITOR` as well

Reply:
I'd say they should all go in the individual man pages, since you'll need them specifically for those commands. `DISABLE_METRICS` might be worth duplicating

Reply:
I'd probably duplicate all of them since people are likely to `man flox` more than they are to `man flox-activate`. If we start amassing a giant list of environment variables I'd say no duplication, but for now there's not that many (user facing) environment variables.

Reply:
Sounds good 👍